### ISSUE TITLE
[bug] Copyright symbol is not an emoji

### ISSUE THREAD

**Issue by rachelgould** [vue-a11y/eslint-plugin-vuejs-accessibility#342]
In updating from `0.7.1` to `1.1.0`, I started seeing a violation for the `vuejs-accessibility/accessible-emoji` rule:
`Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby`. 

It appears that this is being triggered for anywhere that has a copyright symbol, written as `&copy;`. I believe this to be a bug, since screen readers are able to read this character.

**Comment by kddnewton** [comment]
I'm going to close this only because that rule is deprecated and is going to be removed in the next version anyway.
